db/migrate/20160413171549_unique_roles.rb

```diff
@@ -0,0 +1,14 @@
+class UniqueRoles < ActiveRecord::Migration
+  def up
+    UserProjectRole.all.group_by(&:user_id).each do |user, roles|
+      roles.sort_by(&:role_id)[0..-2].each(&:destroy)
+    end
```

Comment:
Also, could you add a comment saying what this is doing. I had to read the code a few times to get what was going on